include/libtorrent/kademlia/dht_storage.hpp

```diff
@@ -70,34 +70,24 @@ namespace dht
 		void reset();
 	};
 
-	struct TORRENT_EXPORT dht_immutable_item
+	struct TORRENT_EXPORT dht_immutable_data
```

Comment:
I think it's a bit confusing to have duplicate structs though, that are so similar and have such similar names. It's not obvious I think what they are used for (at least not from their names)